.github/workflows/web-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Capa Explorer Web tests 
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+      - 'web/explorer/**'
```

Comment:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#example-including-paths